x-pack/platform/plugins/shared/onechat/public/application/hooks/tools/use_execute_tools.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMutation } from '@tanstack/react-query';
+import { ExecuteToolResponse } from '../../../../common/http_api/tools';
+import { useOnechatServices } from '../use_onechat_service';
+
+export interface ExecuteToolParams {
+  toolId: string;
+  toolParams: Record<string, any>;
```

Comment:
I think it's better to use `unknown` than `any`. Registry uses `Record<string, unknown>`